aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -1,126 +1,59 @@
 ---
-title: ASP.NET Core Module
+title: ASP.NET Core Module for ASP.NET Core
 author: tdykstra
-description: Introduces ASP.NET Core Module (ANCM), an IIS module that lets the Kestrel web server use IIS or IIS Express as a reverse proxy server.
+description: Learn how the ASP.NET Core Module is an IIS module that allows the Kestrel web server use IIS or IIS Express as a reverse proxy server.
 manager: wpickett
 ms.author: tdykstra
-ms.custom: H1Hack27Feb2017
-ms.date: 08/03/2017
+ms.custom: mvc
+ms.date: 02/23/2018
 ms.prod: asp.net-core
 ms.technology: aspnet
 ms.topic: article
 uid: fundamentals/servers/aspnet-core-module
 ---
-# Introduction to ASP.NET Core Module
+# ASP.NET Core Module for ASP.NET Core
 
 By [Tom Dykstra](https://github.com/tdykstra), [Rick Strahl](https://github.com/RickStrahl), and [Chris Ross](https://github.com/Tratcher) 
 
-ASP.NET Core Module (ANCM) lets you run ASP.NET Core applications behind IIS, using IIS for what it's good at (security, manageability, and lots more) and using [Kestrel](kestrel.md) for what it's good at (being really fast), and getting the benefits from both technologies at once. **ANCM works only with Kestrel; it isn't compatible with WebListener (in ASP.NET Core 1.x) or HTTP.sys (in 2.x).** 
+The ASP.NET Core Module allows ASP.NET Core apps to run behind IIS in a reverse proxy configuration. IIS provides advanced web app security and manageability features. [Kestrel](xref:fundamentals/servers/kestrel) provides fast request processing. Creating a reverse proxy between Kestrel and IIS provides the benefits of both technologies.
```

Comment:
The performance is actually much worse than directly using Kestrel. Really what you get is a consistent application model across platforms.

Reply:
> a consistent application model across platforms.

... you mean Kestrel does? I thought Tom (or whoever wrote this) was saying that Kestrel is faster than the ASP.NET 4.x runtime at processing requests. I thought he was saying IIS+ANCM+Kestrel is faster than IIS+ASP.NET 4.x.

We could dump the foul (*and confusing* 😵) contrast by dropping the end of the paragraph. The para becomes ...

> The ASP.NET Core Module allows ASP.NET Core apps to run behind IIS in a reverse proxy configuration. IIS provides advanced web app security and manageability features.